.babelrc

```diff
@@ -2,5 +2,9 @@
   "presets": [
     "@babel/env",
     "@babel/react"
-  ]
+  ],
+    "plugins": [
```

Comment:
whats this plugin for? The airbnb linter?

Reply:
I believe it used for only using the 'helpers' that are needed during the build. I havent used it yet and will remove if I dont.